source/_patterns/02-molecules/leads/lead-gallery.twig

```diff
@@ -2,9 +2,18 @@
 
   <div class="c-lead-gallery__top">
 
-    {% include '@molecules/article/figure-with-caption.twig' %}
+    <div class="c-lead-gallery__image js-toggle js-gallery-toggle" data-toggled="c-main" data-prefix="gallery">
+      {%
+        include 'molecules-figure'
+        with {
+          caption: false,
+          credit: false
+        }
+      %}
+    </div>
 
     <div class="c-lead-gallery__text">
+      {% include 'molecules-credit' %}
```

Comment:
the problem with placing the credit here is that 1) it's not a `figcaption` and 2) a `figcaption` must be a direct child of a `figure`. I'm worry about screen readers not being able to properly parse a figure with this structure.

Also, remember that there's no more `lead_slides_dek`, there's only the `caption` and `title`.